### ISSUE TITLE
use TimeControlsNode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/wave-interference#487]
This sims has its own implementation of time controls in TimeControls.js.  It should be using scenery-phet/TimeControlNode -- especially since it's instrumented for PhET-iO.

**Comment by arouinfar** [comment]
Good catch @pixelzoom. Agreed, we should be using TimeControlNode here @samreid.

**Comment by samreid** [comment]
Fixed in the commit.  I just used the default layout, which led to a larger spacing between the step button and the radio buttons, and the radio buttons are somewhat bigger.  Do you think the defaults will work here, or should I specify customization options?

**Comment by arouinfar** [comment]
@samreid  the spacing looks fine to me, but I would prefer to match the radio button size to Top View/Side View. It looks a bit strange to have varying sizes so close together in the sim.
![image](https://user-images.githubusercontent.com/8419308/80653650-e7804900-8a37-11ea-8193-3f2038069a4c.png)

**Comment by samreid** [comment]
Proposed fix committed, it looks like this:

![image](https://user-images.githubusercontent.com/679486/81623274-56529000-93b0-11ea-9823-b361cb2584b3.png)

@arouinfar can you please review?

**Comment by arouinfar** [comment]
Looks great @samreid, thanks!
